x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/layout.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Size } from '../../../../../types';
+import { ViewZoomWidthHeight } from './index.d';
```

Comment:
shouldn't need the `.d` in the name, and I don't think you need to specify `index` too.  Try:

`import { ViewZoomWidthHeight } from '.';`

Reply:
For the record, this doesn't work because it resolves it to `index.js` which exists in the same location, so have to be explicit with `index.d.ts`.   Maybe the file should just be named `types.d.ts` in that case.  Which also matches our other `types.d.ts` file.